src/components/shared/form_fields/RadioField.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-	<fieldset class="form-field form-field-radio" :class="{ 'is-invalid': showError }">
+	<fieldset class="form-field form-field-radio" :class="[ alignment + '-alignment', { 'is-invalid': showError } ]" >
```

Comment:
```suggestion
	<fieldset class="form-field form-field-radio" :class="[ `${alignment}-alignment`, { 'is-invalid': showError } ]" >
```

Just a matter of taste, I'm also in the habit of concatenating, but the popular opinion seems to be that templates are more readable. The [`prefer-template` rule](https://eslint.org/docs/latest/rules/prefer-template) is not active in our linting